jobs/ContinuousTest/Jenkinsfile

```diff
@@ -52,7 +52,14 @@ node{
         }
```

Comment:
Can you please summarize the change key points of your PR ?
it's not small and not easy to review without some guidance